flask-server/avenv/lib/python3.12/site-packages/click/_winconsole.py

```
@@ -0,0 +1,296 @@
+from ctypes import POINTER
+from ctypes import py_object
+from ctypes import Structure
+from ctypes.wintypes import DWORD
+from ctypes.wintypes import HANDLE
+from ctypes.wintypes import LPCWSTR
+from ctypes.wintypes import LPWSTR
+
+from ._compat import _NonClosingTextIOWrapper
+
+assert sys.platform == "win32"
+import msvcrt  # noqa: E402
```

Ruff (0.13.2)

Unused `noqa` directive (non-enabled: `E402`)

Remove unused `noqa` directive

(RUF100)